hackathon_site/registration/migrations/0009_alter_application_q4.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 3.2.5 on 2021-07-31 07:30
```

Comment:
Before this is actually merged, migrations need to be squashed so there's only one:

```bash
$ python manage.py migrate registration 0007
$ rm <path to migration 8>
$ rm <path to migration 9>
$ python manage.py makemigrations
$ python manage.py migrate
```